### ISSUE TITLE
Add SearchSolutions Workshop Series

### ISSUE THREAD

**Issue by potthast** [ir-anthology/ir-anthology#46]
https://irsg.bcs.org/SearchSolutions/2020/sse2020.php

**Comment by najtin** [comment]
Do they publish papers? I couldn't find any on their website. However, they have talks with abstracts, though i am not sure whether they have publicly available recordings. What exactly should i add to the ir-anthology? @potthast

**Comment by potthast** [comment]
We start with Bib entries. And we may contact them to provide us with materials if they have any. It is a workshop in IR, thus relevant for our cause.

**Comment by najtin** [comment]
Okay, just to make sure i understand correctly. We want to add the bib of the presentations of the SearchSolutions Workshop Series to the anthology and maybe be latter link to materials of the presentations. Is that right?

**Comment by potthast** [comment]
It coulde be `@Misc` entries, for example, if there is really no paper: It'd be good to have it listed because of it's long history.

![image](https://user-images.githubusercontent.com/1148967/124485622-0cdfb400-ddad-11eb-8f8c-ffe9937ff605.png)

**Comment by najtin** [comment]
I take that as a yes. :D

**Comment by potthast** [comment]
Yes, I'd say so. It is a special case, of course... Given the more or less unifies web pages, some well-chose copy-paste commands and search and replace on the resulting texts might do the trick comparably quick.
